.github/workflows/go.yml

```diff
@@ -50,6 +50,15 @@ jobs:
           - 1.15.x
           - 1.16.x
           - 1.17.x
+        exclude:
+          - os: macos-latest
+            go-version: 1.12.x
+          - os: macos-latest
+            go-version: 1.13.x
+          - os: macos-latest
+            go-version: 1.14.x
+          - os: macos-latest
+            go-version: 1.15.x
```

Comment:
it seems like 1.15.x appears both in this exclude list (of versions not supported) and also on line 100, where it excludes versions that do have support.

Why is that?